convert-redis-to-gemfire/session-state/spring-session-boot-json/README.md

```diff
@@ -27,20 +27,9 @@ implementation 'org.springframework.session:spring-session-data-redis'
 Replace them with these dependencies:
 
 ```groovy
-ext {
-    set('springGeodeVersion', "1.4.0")
-}
-
 dependencies {
-    implementation 'org.springframework.geode:spring-geode-starter'
-    implementation 'org.springframework.geode:spring-geode-starter-session'
-}
-
-dependencyManagement {
-    imports {
-        mavenBom "org.springframework.geode:spring-geode-bom:${springGeodeVersion}"
+    implementation 'com.vmware.gemfire:spring-boot-3.1-vmware-gemfire-10.0:1.0.0'
```

Comment:
Why was the `spring-geode-starter-session` removed and not replaced with the gemfire equivalent?

Reply:
It was running without it, though I can't find anywhere that we are exposing session from the main dependency. I'll add the session starter.